### PR TITLE
 Add error handler for errors during testOracleQuery request

### PR DESCRIPTION
**Pull request by perfectmak**
If an error occurs from the providers (for example rejecting the transation from metamask), this PR ensure the error is caught and displayed to the user.

### CODE REVIEW

File: src/containers/TestQuery.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Contracts from '../Contracts.js';
 const mapStateToProps = (state, ownProps) => {
   const { error, loading, results, transaction } = state.testQuery;
   const { network } = state.web3;
+  console.log('Error', error && error.error);
```
Comment (eswarasai) [inline src/containers/TestQuery.js]: @perfectmak -- Can you please remove the `console.log()` here? Thanks

**Comment by eswarasai** [comment]
@perfectmak -- Please do `git pull -r upstream master` and push the changes instead of the explicit merge commits. Thanks.

**Comment by perfectmak** [comment]
@eswarasai I've removed the log.

**Comment by pelsasser** [comment]
@eswarasai - can you confirm good to merge? @perfectmak -thanks!  I owe you some ETH for all these commits!

**Pull request merged**